src/braket/pennylane_plugin/translation.py

```diff
@@ -25,18 +26,79 @@
     StateVector,
     Variance,
 )
+from braket.schema_common import BraketSchemaBase
 from pennylane.operation import Observable, ObservableReturnTypes, Operation
 
 from braket.pennylane_plugin.ops import PSWAP, XY, YY, CPhaseShift00, CPhaseShift01, CPhaseShift10
 
-
-def supported_operations() -> FrozenSet[str]:
-    """Returns the operations supported by the plugin.
+BRAKET_TO_PENNYLANE_OPERATIONS = {
```

Comment:
You'll also want to add supported [observables](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.Device.html#pennylane.Device.observables)